packages/web/src/pages/sign-up-page/components/SocialButton.tsx

```diff
@@ -0,0 +1,41 @@
+import {
```

Comment:
This feels like we should promote this to Harmony or somewhere more reusable, would love some feedback here

Reply:
Yeah if we wanted to be "atomic" with it we could consider our base components our "atoms" and "pre-made extensions of our base components" our "molecules". 
Idk the best term to name it tbh, maybe would also consider "extensions" or "composites/compounds" or something 🤔

Reply:
I kinda like the idea of separating "core" or "base" components from the "premade extensions" though